src/inc/til/winrt.h

```diff
@@ -8,12 +8,12 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
     template<typename T>
     struct property
     {
-        explicit constexpr property(auto&&... args) :
+        explicit constexpr property(auto&&... args) noexcept(std::is_nothrow_constructible_v<T, decltype(args)...>) :
             _value{ std::forward<decltype(args)>(args)... } {}
 
         property& operator=(const property& other) = default;
 
-        T operator()() const noexcept
+        T operator()() const noexcept(std::is_nothrow_copy_constructible<T>::value)

```

Comment:
Ah, I had it in my head that this was from `V` to `T`. But it's actually from `T` to `T`, so that's OK.

@PankajBhojwani what did you see when you tried to return `const T&`? That definitely should work...

Reply:
![image (1)](https://github.com/user-attachments/assets/c583f89c-4ec4-4f2f-8ab2-74a64ec55280)

Reply:
@lhecker confirmed that we can't do the `const T&` route as well

Reply:
thanks!